native/com_wolfssl_WolfSSLSession.c

```diff
@@ -6154,8 +6577,18 @@ int NativeSSLIOSendCb(WOLFSSL *ssl, char *buf, int sz, void *ctx)
     }
 
     /* Detect if we should use ByteBuffer or byte[] I/O callbacks */
-    if ((*jenv)->CallBooleanMethod(jenv, (jobject)(*g_cachedSSLObj),
-            g_isByteBufferIOSendCallbackSet)) {
+    jboolean useByteBuffer = (*jenv)->CallBooleanMethod(jenv,
```

Comment:
Can you declare this `jboolean useByteBuffer` at the top of the function with other var declarations?